exporter/exporter.go

```diff
@@ -1,4 +1,4 @@
-// mongodb_exporter
+// mnogo_exporter
```

Comment:
```suggestion
// mongodb_exporter
```

Reply:
@percona-csalguero Could you check in other places and rename?

Reply:
Done in last commit